src/platform/forms-system/src/js/widgets/SelectWidget.jsx

```diff
@@ -42,10 +42,12 @@ function SelectWidget({
       disabled={disabled}
       readOnly={readonly}
       onBlur={event => {
+        // eslint-disable-next-line sonarjs/no-extra-arguments
         const newValue = getValue(event, multiple);
         onBlur(id, processValue(schema, newValue));
       }}
       onChange={event => {
+        // eslint-disable-next-line sonarjs/no-extra-arguments
```

Comment:
Since it's not clear why the function is taking 2 params instead of 1, this rule was disabled to be in compliance with the activation of the rule. I disabled it to prevent that the changes might break the existing form system.